### PR TITLE
Translated Portals

### PR DESCRIPTION
**Pull request by lunmay**

### CODE REVIEW

File: content/docs/portals.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
 
-Portals provide a first-class way to render children into a DOM node that exists outside the DOM hierarchy of the parent component.
+Portal menyediakan cara utama untuk me-*render* anak ke dalam simpul DOM yang berada di luar hierarki komponen induk.
 
 ```js
 ReactDOM.createPortal(child, container)
 ```
 
-The first argument (`child`) is any [renderable React child](/docs/react-component.html#render), such as an element, string, or fragment. The second argument (`container`) is a DOM element.
+Argumen pertama (`child`) berupa [anak React yang bisa di-*render*](/docs/react-component.html#render), misalnya sebuah elemen, string, atau *fragment*. Argumen kedua (`container`) merupakan elemen DOM.
```
Comment (regalius) [inline content/docs/portals.md]: string -> *string*

File: content/docs/portals.md [hunk 2]
```diff
@@ -27,34 +27,35 @@ render() {
...
+> Saat bekerja dengan portal, perhatikan bahwa [mengelola fokus papan ketik](/docs/accessibility.html#programmatically-managing-focus) menjadi sangat penting.
 >
-> For modal dialogs, ensure that everyone can interact with them by following the [WAI-ARIA Modal Authoring Practices](https://www.w3.org/TR/wai-aria-practices-1.1/#dialog_modal).
+> Untuk dialog modal, pastikan semua pihak bisa berinteraksi dengannya dengan mengikuti [WAI-ARIA Modal Authoring Practices](https://www.w3.org/TR/wai-aria-practices-1.1/#dialog_modal).
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/yzMaBd)
+[**Coba di CodePen**](https://codepen.io/gaearon/pen/yzMaBd)
 
-## Event Bubbling Through Portals {#event-bubbling-through-portals}
+## *Event Bubbling* lewat Portal {#event-bubbling-through-portals}
```
Comment (regalius) [inline content/docs/portals.md]: Loanwords in the heading doesn't have to be translated 😄

File: content/docs/portals.md [hunk 3]
```diff
@@ -27,34 +27,35 @@ render() {
...
 
-Even though a portal can be anywhere in the DOM tree, it behaves like a normal React child in every other way. Features like context work exactly the same regardless of whether the child is a portal, as the portal still exists in the *React tree* regardless of position in the *DOM tree*.
+Walau sebuah portal bisa berada di mana saja dalam pohon DOM, portal tersebut berperilaku seperti halnya anak React yang normal. Fitur seperti *context work* bekerja sama persis tanpa tergantung apakah sebuah anak adalah sebuah portal, karena portal masih berada dalam *pohon React* tanpa memandang posisinya dalam *pohon DOM*.
```
Comment (regalius) [inline content/docs/portals.md]:
```suggestion
Walau sebuah portal bisa berada di mana saja dalam pohon DOM, portal tersebut berperilaku seperti halnya anak React yang normal. Fitur seperti *context* bekerja sama persis tanpa tergantung apakah sebuah anak adalah sebuah portal, karena portal masih berada dalam *pohon React* tanpa memandang posisinya dalam *pohon DOM*.
```

I think it's typo 😄

File: content/docs/portals.md (hunk 1 above)
Comment (grikomsn) [inline content/docs/portals.md]:
```suggestion
Argumen pertama (`child`) berupa [anak React yang bisa di-*render*](/docs/react-component.html#render), misalnya sebuah elemen, *string*, atau *fragment*. Argumen kedua (`container`) merupakan elemen DOM.
```

File: content/docs/portals.md (hunk 2 above)
Comment (grikomsn) [inline content/docs/portals.md]:
```suggestion
## Event Bubbling lewat Portal {#event-bubbling-through-portals}
```

File: content/docs/portals.md [hunk 4]
```diff
@@ -1,24 +1,24 @@
...
-## Usage {#usage}
+## Penggunaan {#usage}
 
-Normally, when you return an element from a component's render method, it's mounted into the DOM as a child of the nearest parent node:
+Umumnya saat Anda mengembalikan sebuah elemen dari *method* *render* komponen, elemen tersebut dipasang ke DOM sebagai anak pada simpul induk terdekat:
 
 ```js{4,6}
 render() {
-  // React mounts a new div and renders the children into it
+  // React memasang div baru dan me-render anaknya kepadanya
```
Comment (grikomsn) [inline content/docs/portals.md]:
```suggestion
  // React memasang div baru dan me-*render* anaknya kepadanya
```

**Pull request merged**